OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -314,7 +317,7 @@ function ShowWindow()
 function Refresh(bool bActivatedByUser)
 {
 	local R6WindowListServerItem CurServer;
-
+	

```

Comment:
extra whitespace

Reply:
My editor auto-adds tabs when you press enter so you're at the same indent level on the next line, that's all. :)